src/Uno.Extensions.Navigation.Toolkit/Uno.Extensions.Navigation.Toolkit.WinUI.csproj

```diff
@@ -4,7 +4,7 @@
 	<PropertyGroup>
 		<AssemblyName>Uno.Extensions.Navigation.Toolkit.UI</AssemblyName>
 		<NoWarn>$(NoWarn);CA1416</NoWarn>
-
+		<WindowsSdkPackageVersion>10.0.19041.57</WindowsSdkPackageVersion>
```

Comment:
Why is this still required?

Reply:
I was getting this error

> Error CS1705: Assembly 'Uno.Toolkit.WinUI' with identity 'Uno.Toolkit.WinUI, Version=255.255.255.255, Culture=neutral, PublicKeyToken=null' uses 'WinRT.Runtime, Version=2.2.0.0, Culture=neutral, PublicKeyToken=99ea127f02d97709' which has a higher version than referenced assembly 'WinRT.Runtime' with identity 'WinRT.Runtime, Version=2.1.0.0, Culture=neutral, PublicKeyToken=99ea127f02d97709'

Which was addressed in other repos like so: https://github.com/unoplatform/uno.csharpmarkup/commit/9dc1a656d5d53c9aecceb72b27c08fd5339c4228#diff-9da24614831c308827a1ae533ffea392c97638c261dd42bd0f5226baa136d16eR45

I may have overdone it and set it everywhere just in case. Will see if this is indeed needed

Reply:
@jeromelaban correct me if I'm wrong but this was fixed in Uno where this property is no longer required. Do we need to update to latest dotnet sdk in the build? or is there an alternative solution.

Reply:
@nickrandolph I see this as part of the migration steps for Uno 6: https://platform.uno/docs/articles/migrating-to-uno-6.html#winappsdk-17-considerations

Perhaps it's only needed since we are target net8-windows still? Not sure